src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js

```diff
@@ -115,8 +148,93 @@ const Body = BaseForm.extend({
   }
 });
 
+const Footer = AuthenticatorFooter.extend({
+  props: {
+    enabled: ['boolean', false]
+  },
+
+  events: {
+    'click .js-cant-verify': function(e) {
+      e.preventDefault();
+      if (!this.props.enabled) {
+        return;
+      }
+      this.toggleLinks(e);
+    },
+  },
+
+  initialize: function() {
+    let links = _.resultCtx(this, 'links', this);
+    const footerInfo = _.resultCtx(this, 'footerInfo', this);
+    const hasBackToSignInLink = _.resultCtx(this, 'hasBackToSignInLink', this);
+
+    if (!Array.isArray(links)) {
+      links = [];
+    } else {
+      links = links.filter(l => $.isPlainObject(l));
+    }
+
+    if (this.options.appState.shouldShowSignOutLinkInCurrentForm(
+      this.options.settings.get('features.hideSignOutLinkInMFA') ||
+      this.settings.get('features.mfaOnlyFlow')) && hasBackToSignInLink) {
+      links = links.concat(getBackToSignInLink(this.options.settings));
+    }
+
+    links.forEach(link => {
+      this.add(Link, {
+        options: link,
+      });
+      if (link.name === 'cant-verify') {
+        if (this.options.appState.get('app') && this.options.appState.get('app').name === 'Okta_Authenticator') {
+          this.add(CantVerifyInfoOVEnrollmentFlowTemplate);
+        } else {
+          this.add(CantVerifyInfoVerifyFlowTemplate);
+        }
+      }
+    });
+
+    if (footerInfo) {
+      this.add(View.extend({
+        className: 'footer-info',
+      }));
+
+      this.add(footerInfo, '.footer-info');
+    }
+
+    this.listenTo(this.props.enabled, 'change:enabled', function(model, enable) {
+      this.$('.link').toggleClass('o-form-disabled', !enable);
+    });
+  },
+
+  links: function() {
+    let links = AuthenticatorFooter.prototype.links.apply(this, arguments);
```

Comment:
can be a `const` since this is an array and you are just adding a new object to it, not changing the reference to it.